test/test_configuration.py

```diff
@@ -296,21 +300,53 @@ class Config(self.UrlApiConfig):
 
             Config(_create_endpoints())
 
+        with self.assertRaises(RestClientConfigurationError):
+
+            class Config(self.UrlApiConfig):
+                default_timeout = "error"
+
+            Config(_create_endpoints())
+
+        with self.assertRaises(RestClientConfigurationError):
+
+            class Config(self.UrlApiConfig):
+                default_timeout = (3,)
+
+            Config(_create_endpoints())
+
+        with self.assertRaises(RestClientConfigurationError):
+
+            class Config(self.UrlApiConfig):
+                default_timeout = (None, -3.0)
+
+            Config(_create_endpoints())
+
     def test_good_defaults(self):
```

Comment:
I propose to split up `test_good_defaults` as it tests 3 different cases:
- if one test case fails the remaining ones aren't tested anymore;
- with 3 separate tests you can have more descriptive test names.

Reply:
Will adjust the code